Devdocs 3434 (#1367)
bigcommerce/dev-docs: docs/api-docs/building-checkouts/checkout-customizability.md

</div> 

> BigCommerce provides significant levels of checkout customizability. If you just want to modify the theme by overriding the style of BigCommerce’s Optimized One-Page Checkout, please follow [this article about restyling one-page checkouts](https://developer.bigcommerce.com/stencil-docs/customizing-checkout/optimized-one-page-checkout). However, if you want to build a custom checkout, there are two paths that a developer can choose to use.

Reviewer: ```suggestion
BigCommerce provides significant levels of checkout customizability. If you just want to modify the theme by overriding the style of BigCommerce’s Optimized One-Page Checkout, please see [Restyle Optimized One-Page Checkout](https://developer.bigcommerce.com/stencil-docs/customizing-checkout/optimized-one-page-checkout). However, if you want to build a custom checkout, there are two paths that a developer can choose to use.
```
Suggest changing the link to the article title
